nms/app/packages/magmalte/app/views/subscriber/SubscriberOverview.js

```diff
@@ -174,16 +189,83 @@ function JsonDialog(props: Props) {
   );
 }
 
-function SubscriberTableRaw(props: WithAlert) {
+function Subscribers() {
+  const [refresh, setRefresh] = useState(true);
   const classes = useStyles();
-  const {history, relativeUrl} = useRouter();
+
+  return (
+    <div className={classes.dashboardRoot}>
+      <Grid container spacing={4}>
+        <Grid item xs={12}>
+          <Grid item xs={12} container>
+            <Grid
+              container
+              alignItems="center"
+              className={classes.cardTitleRow}>
+              <Grid item xs>
+                <Grid container alignItems="center">
+                  <PeopleIcon className={classes.cardTitleIcon} />
+                  <Text variant="body1">{TITLE}</Text>
+                </Grid>
+              </Grid>
+              <Grid item>
+                <Grid
+                  container
+                  justify="flex-end"
+                  alignItems="center"
+                  spacing={2}>
+                  <Grid item>
+                    <AddSubscriberButton />
+                  </Grid>
+                  <Grid item>
+                    <FormControlLabel
+                      value="top"
+                      control={
+                        <SwitchButton
```

Comment:
Can you move this before AddSubscriberButton ? Also can we use the Autorefresh checked button consistently here as well or modify that page to use the switch. I''m referring to the Gateway Event Table and GatewayLog Table